fr-swc/jw300-baseline/baseline.ipynb

```diff
@@ -0,0 +1,4443 @@
+{
```

Comment:
<p>We should save these 643 in the global test set</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/masakhane-io/masakhane/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Agreed!

Reply:
This is done @jaderabbit !!🤦🏾‍♂️